pkg/vpcmodel/semanticDiff.go

```diff
@@ -176,14 +233,14 @@ func getDiffType(src, srcInOther, dst, dstInOther VPCResourceIntf) DiffType {
 // EnhancedString ToDo: likely the current printing functionality will no longer be needed once the grouping is added
 // anyways the diff print will be worked on before the final merge
 
-func (diff *DiffBetweenSubnets) String() string {
-	return diff.subnet1Subtract2.EnhancedString(true) +
-		diff.subnet2Subtract1.EnhancedString(false)
+func (diff *diffBetweenCfgs) String() string {
+	return diff.cfg1ConnRemovedFrom2.EnhancedString(diff.diffAnalysis, true) +
+		diff.cfg2ConnRemovedFrom1.EnhancedString(diff.diffAnalysis, false)
 }
 
-func (subnetDiff *SubnetsDiff) EnhancedString(thisMinusOther bool) string {
+func (connDiff *connectivesDiff) EnhancedString(diffAnalysis diffAnalysisType, thisMinusOther bool) string {
```

Comment:
Answer to both questions - it seemed more consistent with the existing `EnhancedString` funcs.
Also, unlike String() funcs it receives parms.
I unexported it; but are you sure you want to rename to String? in this case would you like it with a capital in which case it will be exported?

Reply:
ok, done